src/strategies/roundrobin.js

```diff
@@ -0,0 +1,16 @@
+
+module.exports = class RoundRobinStrategy {
+
+	constructor() {
+		this.counter = 0;
+	}
+
```

Comment:
And maybe an abstract BaseStrategy class which implement this `init`, what you can override in your custom strategies :)